client/jetpack-cloud/sections/partner-portal/revoke-license-dialog/style.scss

```diff
@@ -27,10 +27,10 @@
 	&__heading {
 		margin: -16px -16px 16px;
 		padding: 9px 16px;
-		font-size: 1.25rem;
+		font-size: 1.125rem; /* stylelint-disable */
```

Comment:
Why the disable? Is that size disallowed?

Reply:
Yes, stylelint complains about that size.